rapids/README.md

```diff
@@ -15,8 +15,9 @@ action supports Dask and Spark runtimes for RAPIDS on
 initialization actions in production.
 
 This initialization action will install RAPIDS on Dataproc for either Spark or
-Dask. RAPIDS for Spark is supported on Dataproc 1.5 (Spark 2.4) or Dataproc
-2.0+ (Spark 3.0)+. RAPIDS for Dask is only supported on Dataproc 2.0+.
+Dask. RAPIDS for Spark is supported on Dataproc 2.0+ (Spark 3.0)+. 
+Spark with XGBoost is support on Dataproc 1.5+ with Spark 2.4.
```

Comment:
Can we change this to "RAPIDS SQL Accelerator" / the official name for the Spark / RAPIDS plugin, for slightly improved clarity?

Reply:
Fixed the naming in a number of places